Makefile

```diff
@@ -161,13 +245,21 @@ ifneq ($(HAS_SERVER),)
 	rm -fr server/dist
 endif
 ifneq ($(HAS_WEBAPP),)
-	rm -fr webapp/.npminstall
 	rm -fr webapp/junit.xml
 	rm -fr webapp/dist
 	rm -fr webapp/node_modules
 endif
 	rm -fr build/bin/
 
+## Sync directory with a starter template
+sync:
+ifndef STARTERTEMPLATE_PATH
+	@echo STARTERTEMPLATE_PATH is not set.
+	@echo Set STARTERTEMPLATE_PATH to a local clone of https://github.com/mattermost/mattermost-plugin-starter-template and retry.
+	@exit 1
+endif
+	cd ${STARTERTEMPLATE_PATH} && go run ./build/sync/main.go ./build/sync/plan.yml $(PWD)
```

Comment:
It looks like `./build/sync` didn't made it into this PR. It's fine, but I'm wondering why?

Reply:
Ooops, forgotten this one! Added in https://github.com/mattermost/mattermost-plugin-custom-attributes/pull/71/commits/d9b881374e7058f37c56fc3d0b87fecf999b25f3